wiki/People/The_Team/Global_Moderation_Team/th.md

```diff
@@ -3,117 +3,116 @@ tags:
   - GMT
   - moderator
   - moderators
-no_native_review: true
-outdated: true
-outdated_since: fde71096eb656503d1e9d3b85c8ce872de533837
 ---
 
 # ทีมผู้ดูแล
 
-**Global Moderation Team (ทีมผู้ดูแล)** (เดิมชื่อ Global Mod Team, Administrators หรือ GreenBAT) โดยทั่วไปเรียกว่า **GMT** เป็นสมาชิกของชุมชนที่มุ่งเน้นในการดูแลผู้เล่นเป็นหลัก และ ประเด็นที่เกี่ยวข้องกับทีม และ ชุมชนโดยรวม
+**Global Moderation Team (ทีมผู้ดูแล)** (เดิมชื่อ Global Mod Team, Administrators หรือ GreenBAT) โดยทั่วไปเรียกว่า **GMT** เป็นสมาชิกของชุมชนที่มุ่งเน้นในการดูแลผู้เล่นเป็นหลัก ประเด็นที่เกี่ยวข้องกับทีม และชุมชนโดยรวม
+
+**Global Moderator ทุกคน ได้สาบานที่จะทำตาม [จรรยาบรรณผู้ช่วยเหลือ](/wiki/Contributor_Code_of_Conduct) นอกเหนือจาก[กฎชุมชน](/wiki/Rules)**
+
+หากคุณพบเห็นสมาชิกคนใดของ Global Moderation Team มีพฤติกรรมที่ไม่เหมาะสม โปรดติดต่อ [abuse@ppy.sh](mailto:abuse@ppy.sh) โดยทันที พร้อมข้อมูลมากที่สุดเท่าที่เป็นไปได้ สำหรับรายละเอียดเพิ่มเติม ดูที่ [รายงานการละเมิด](/wiki/Reporting_Bad_Behaviour/Abuse)
 
 ## หน้าที่และความรับผิดชอบ
 
-ทีมผู้ดูแลมีหน้าที่รับผิดชอบด้านสวัสดิการของการแชท ฟอรัม และ ดูแลการมีปฏิสัมพันธ์ภายในชุมชน พวกเขาควรได้รับการติดต่อด้วยคำถามใด ๆ ที่เกี่ยวข้องกับแง่มุมการกลั่นกรองของเกมซึ่งรวมถึง (แต่ไม่จำกัดเฉพาะ) ปัญหาเกี่ยวกับการแชท (เช่น ความเงียบ หรือ การล่วงละเมิด) และฟอรัม (เช่น การสแปม หรือ ความคิดเห็นที่ไม่เหมาะสม)
+ทีมผู้ดูแลมีหน้าที่รับผิดชอบด้านสวัสดิการของการแชท ฟอรัม และ ดูแลการมีปฏิสัมพันธ์ภายในชุมชน พวกเขาควรได้รับการติดต่อด้วยคำถามใด ๆ ที่เกี่ยวข้องกับแง่มุมการกลั่นกรองของเกมซึ่งรวมถึง (แต่ไม่จำกัดเฉพาะ) ปัญหาเกี่ยวกับการแชท (เช่น ความเงียบ หรือการล่วงละเมิด) และฟอรัม (เช่น การสแปม หรือความคิดเห็นที่ไม่เหมาะสม)
 
 ## การเป็นสมาชิกทีมผู้ดูแล
 
-การเป็นสมาชิกของทีมผู้ดูแลนั้นต้องการการสนับสนุนที่มีจำนวนมาก สมาชิกที่มีศักยภาพมักจะสังเกตเห็นว่าพวกเขาเต็มใจที่จะช่วยเหลือผู้อื่น กิจกรรมภายในชุมชน และ พฤติกรรมที่ดี ความสามารถในการพูดภาษาอื่นที่ไม่ใช่ภาษาอังกฤษ เป็นที่ต้องการอย่างยิ่ง ผู้สมัครจะถูกพูดคุยภายในทีมเพื่อพิจารณาการเพิ่มศักยภาพของพวกเขาในฐานะสมาชิกในทีม
+การมาเป็นสมาชิกของทีมผู้ดูแลนั้น จำเป็นต้องมีการสนับสนุนอย่างมาก สมาชิกที่มีศักยภาพมักจะถูกสังเกตจากที่ว่าพวกเขาเต็มใจที่จะช่วยเหลือผู้อื่น มีกิจกรรมภายในชุมชน และ พฤติกรรมที่ดี ความสามารถในการพูดภาษาอื่นที่นอกเหนือจากภาษาอังกฤษเป็นที่ต้องการอย่างยิ่ง ผู้สมัครจะถูกพูดคุยภายในทีมเพื่อพิจารณาศักยภาพของพวกเขาในฐานะสมาชิกทีม
```

Comment:
```suggestion
การมาเป็นสมาชิกของทีมผู้ดูแลนั้น จำเป็นต้องมีการสนับสนุนอย่างมาก สมาชิกที่มีศักยภาพมักจะถูกสังเกตจากที่ว่าพวกเขาเต็มใจที่จะช่วยเหลือผู้อื่น มีกิจกรรมภายในชุมชนและ พฤติกรรมที่ดี ความสามารถในการพูดภาษาอื่นที่นอกเหนือจากภาษาอังกฤษนั้นเป็นที่ต้องการอย่างยิ่ง ผู้สมัครจะถูกหารือกันภายในทีมเพื่อพิจารณาศักยภาพของพวกเขาในฐานะสมาชิกทีม
```

Reply:
I've reworded the first part of the suggestion. Applied the second part in the newest commit